### PR TITLE
[RTC-279] Allow to create tracks immediately after onConnect callback

### PR DESCRIPTION
**Pull request by graszka22**
For consistency with the web version. Merge #40 first

### CODE REVIEW

File: Sources/MembraneRTC/MembraneRTC.swift [hunk 1]
```diff
@@ -195,9 +195,7 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
 
             localEndpoint = localEndpoint.withTrack(trackId: videoTrack.rtcTrack().trackId, metadata: metadata)
 
-            if state == .connected {
```
Comment (skyman503) [inline Sources/MembraneRTC/MembraneRTC.swift:198]: Why was this removed?

Comment (graszka22) [reply Sources/MembraneRTC/MembraneRTC.swift]: Try to connect two ios devices with this line. They won't see each other's tracks.
It's because the `state` refers to peer connection state and not state of the app in general. It's necessary to send `renegotiate` message before peer connection is established in order to start the process of establishing peer connection.

**Review by skyman503 — APPROVED** [review]
